### DISCUSSION TITLE
Example for starship prompt

### DISCUSSION THREAD

**Discussion by kakawait** [marlonrichert/zsh-snap#73] [Q&A] [answered]
I'm trying to migrate to _no manager_ simple zsh config to znap (thanks to making that plugin manager fast and easy. I've tried zinit but too complex for me).

My prompt is https://starship.rs/

And my existing `.zshrc` contains

```
if (which starship > /dev/null) then
  eval "$(starship init zsh)"
fi
```

Do you have idiomatic way to load this prompt with znap?

**Comment by marlonrichert** [comment, marked as answer]
```zsh
znap eval starship 'starship init zsh --print-full-init'
znap prompt
```

**Reply by kakawait** [reply]
I was not far from that but lack of starship option and didn't use prompt after my eval.
Thank you
